app/stores/interactive-filters.ts

```diff
@@ -58,9 +58,8 @@ export const useInteractiveFiltersStore = create<
     },
     removeCategory: (category: string) => {
       return set((state) => {
-        const { categories } = state;
-        delete categories[category];
-        return { ...state, categories };
+        delete state.categories[category];
+        return { ...state, categories: { ...state.categories } };
```

Comment:
`return {...state, categories: omit(state.categories, category) )` could be a little clearer.

Reply:
I wrote this down and will tackle this in next PR 👌